iterator_test.go

```diff
@@ -2737,3 +2737,176 @@ func BenchmarkSeekPrefixTombstones(b *testing.B) {
 		iter.SeekPrefixGE(seekKey)
 	}
 }
+
+func runBenchmarkQueueWorkload(b *testing.B, deleteRatio float32, initOps int, valueSize int) {
+	const queueCount = 8
+	// These should be large enough to assign a unique key to each item in the
+	// queue.
+	const maxQueueIDLen = 1
+	const maxItemLen = 7
+	const maxKeyLen = maxQueueIDLen + 1 + maxItemLen
+	queueIDKeyspace := testkeys.Alpha(maxQueueIDLen)
+	itemKeyspace := testkeys.Alpha(maxItemLen)
+	key := make([]byte, maxKeyLen+1+len(fmt.Sprint(b.N)))
```

Comment:
what's the purpose of the `+1+len(fmt.Sprint(b.N))` part? the iteration number is already going into the `itemKeyspace` part, right?

Reply:
This was left behind from an earlier way I was writing keys - I've changed to just be `maxItemLen`